examples/python/human_pose_tracking/main.py

```diff
@@ -190,6 +190,23 @@ def get_downloaded_path(dataset_dir: Path, video_name: str) -> str:
     return str(destination_path)
 
 
+def get_blueprint() -> rrb.BlueprintLike:
```

Comment:
I haven't seen this anywhere else yet done with a getter utility. I don't mind per se, but we shouldn't set a random precedence imho.

Reply:
Yeah I did that in the release checklist. I'll change it for consistency, but I do find having layout code right in the middle of administrative CLI stuff is kind of weird :)

Reply:
we might need to talk this out and do a quick pass at the end then again